docs/outputs.rst

```diff
@@ -23,7 +23,8 @@ ts_OC.nii                 Optimally combined time series.
 dn_ts_OC.nii              Denoised optimally combined time series.
 lowk_ts_OC.nii            Combined time series from rejected components.
 midk_ts_OC.nii            Combined time series from "mid-k" rejected components.
-hik_ts_OC.nii             High-kappa time series.
+hik_ts_OC.nii             Time series of BOLD-like accepted components constructed
```

Comment:
Can we add that this is missing thermal noise, here ?

Otherwise I feel like it seems as though we're recommending this output, when I personally (and I think @handwerkerd said something to this effect, too) would not recommend it for most use cases.

Reply:
Absolutely.